articles/search/cognitive-search-create-custom-skill-example.md

```diff
@@ -46,23 +46,30 @@ Visual Studio creates a project and in it a class that contains boilerplate code
 Now, replace all of the content of the file *Function1.cs* with the following code:
 
 ```csharp
+using System;
+using System.Net.Http;
+using System.Collections.Generic;
+using System.Threading.Tasks;
 using System.IO;
+using System.Text;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Azure.WebJobs;
 using Microsoft.Azure.WebJobs.Extensions.Http;
 using Microsoft.AspNetCore.Http;
 using Microsoft.Azure.WebJobs.Host;
 using Newtonsoft.Json;
-using System.Net.Http;
-using System.Collections.Generic;
-using System.Threading.Tasks;
-using System.Text;
 
 namespace TranslateFunction
 {
     // This function will simply translate messages sent to it.
     public static class Function1
     {
+
+        string path = "https://api.cognitive.microsofttranslator.com/translate?api-version=3.0";
+
+        // NOTE: Replace this example key with a valid subscription key.
+        string key = "064d8095730d4a99b49f4bcf16ac67f8";
```

Comment:
Having a legitimate looking key will cause problems in security reviews.  Can you change it to:
string key = "<put a valid api-key here>"